guides/common/modules/proc_removing-all-content-view-environments-from-an-activation-key.adoc

```diff
@@ -0,0 +1,22 @@
+[id="removing-all-content-view-environments-from-an-activation-key"]
+= Removing all content view environments from an activation key
+
+You can remove all content view environments from an activation key by doing either of the following:
+
+.CLI procedure
+. Remove all content view environments from the activation key by passing an empty array:
```

Comment:
```suggestion
* Remove all content view environments from the activation key by passing an empty array:
```
These are two options of the same action, not two steps. If it's documented in Hammer CLI help, I don't think it's necessary to include both. But if you want to keep them, let's make it an itemized list.